modules/mod_acl_user_groups/templates/_admin_acl_rule_row.tpl

```diff
@@ -13,6 +13,10 @@
     {% endif %}
     <form id="{{ #ruleform }}" class="row acl-rule-row acl-rule-{{ rule.id }} {% if rule.is_block %}is_block{% endif %}" method="post" action="postback">
 
+        {% if rule.managed_by %}
+            <fieldset disabled="disabled">
+        {% endif %}
```

Comment:
I just disable the inputs in the interface, which is of course very soft deletion protection. Do we want to harden this in the `m_acl_rule` module?